code/game/objects/items/devices/scanners.dm

```diff
@@ -267,20 +267,24 @@ REAGENT SCANNER
 			to_chat(usr, "The scanner will now perform a basic analysis.")
 
 /obj/item/device/healthanalyzer/improved //reports bone fractures, IB, quantity of beneficial reagents in stomach; also regular health analyzer stuff
-	name = "advanced health analyzer"
+	name = "enhanced health analyzer"

```

Comment:
This one is named "enhanced", but the one with "enhanced" in the path is "phasic"...?

Reply:
good spot